crates/solidity/testing/snapshots/bindings_output/yul/try_params/input.sol

```diff
@@ -0,0 +1,13 @@
+interface Service { function test() external returns (uint); }
+
+contract Test {
```

Comment:
+1 on test focus. For example for this test, I suggest removing things like `Service` and possibly `add()`. This asserts only the intended functionality, and reduces noise in reviews. e.g.:

```solidity
try true returns (uint v) {
    assembly {
        v := 0
    }
} catch {
}
```

Reply:
What I don't like about that reduction is that it doesn't work on Remix, ie. it's not valid Solidity, because the `try` expression needs to be an external call. But `try this.test()` works.